src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/cdi/ManagedBeanDiagnosticsCollector.java

```diff
@@ -167,12 +162,10 @@ else if (INJECT_FQ_NAME.equals(annotation))
                 }
 
                 if (isProducerMethod && methodScopes.size() > 1) {
-                    List<String> diagnosticData = methodScopes.stream().map(annotation -> getSimpleName(annotation))
-                            .collect(Collectors.toList()); // convert to simple name
-                    diagnosticData.add(PRODUCES);
+                    methodScopes.add(PRODUCES);
```

Comment:
Ensure that all names in the list are qualified names.

Reply:
Done